packages/dd-trace/test/profiling/config.spec.js

```diff
@@ -136,4 +144,55 @@ describe('config', () => {
 
     expect(exporterUrl).to.equal(expectedUrl)
   })
+
+  it('should disable OOM heap profiler by default', () => {
+    const config = new Config()
+    expect(config.oomMonitoring).to.deep.equal({
+      enabled: false,
+      heapLimitExtensionSize: 0,
+      maxHeapExtensionCount: 0,
+      exportStrategies: [],
+      exportCommand: undefined
+    })
+  })
+
+  it('should support OOM heap profiler configuration', () => {
+    process.env = {
+      DD_PROFILING_EXPERIMENTAL_OOM_MONITORING_ENABLED: 'false'
+    }
+    const config = new Config({})
+
+    expect(config.oomMonitoring).to.deep.equal({
+      enabled: false,
+      heapLimitExtensionSize: 0,
+      maxHeapExtensionCount: 0,
+      exportStrategies: [],
+      exportCommand: undefined
+    })
+  })
+
+  it('should support OOM heap profiler configuration', () => {
+    process.env = {
+      DD_PROFILING_EXPERIMENTAL_OOM_MONITORING_ENABLED: '1',
+      DD_PROFILING_EXPERIMENTAL_OOM_HEAP_LIMIT_EXTENSION_SIZE: '1000000',
+      DD_PROFILING_EXPERIMENTAL_OOM_MAX_HEAP_EXTENSION_COUNT: '2',
+      DD_PROFILING_EXPERIMENTAL_OOM_EXPORT_STRATEGIES: 'process,interrupt,async,interrupt'
+    }
+
+    const config = new Config({})
+
+    expect(config.oomMonitoring).to.deep.equal({
+      enabled: true,
+      heapLimitExtensionSize: 1000000,
+      maxHeapExtensionCount: 2,
+      exportStrategies: ['process', 'interrupt', 'async', 'interrupt'],
```

Comment:
I presume pass-through of duplicate values is a desired feature?

Reply:
Nice suggestion, I will add de-duplication of strategies